ibis/backends/risingwave/__init__.py

```diff
@@ -24,6 +24,55 @@
     import pyarrow as pa
 
 
+valid_data_formats = {
+    "PLAIN",
+    "DEBEZIUM",
+    "MAXWELL",
+    "CANAL",
+    "UPSERT",
+    "DEBEZIUM_MONGO",
+}
+valid_encode_formats = {"CSV", "JSON", "AVRO", "PROTOBUF", "BYTES"}
+
+
+def validate_data_format(format_str: str):
+    if format_str is None:
+        return
+    upper = format_str.upper()
+    if upper not in valid_data_formats:
+        raise ValueError(
+            f"Invalid data format: {format_str}. Allowed formats are {', '.join(valid_data_formats)}"
+        )
+
+
+def validate_encode_format(format_str: str):
```

Comment:
Is all this validation necessary? What if a new data format or encoding is added? Can't we let this fall through to the driver to avoid kneecapping users who upgrade their risingwave installation?

Reply:
Make sense. removed.